ui/src/table.tsx

```diff
@@ -208,7 +217,51 @@ const
     const line = JSON.stringify(row)
     return line.substr(1, line.length - 2)
   }).join('\n'),
-  DataTable = ({ model: m, onFilterChange, items, filteredItems, selection, selectedFilters, isMultiple, isSearchable, groups, sort, reset }: DataTable) => {
+  ContextualMenu = ({ onFilterChange, col, listProps, selectedFiltersRef, setFiltersInBulk }: ContextualMenuState) => {
+    const
+      isFilterChecked = (data: S, key: S) => !!selectedFiltersRef.current && selectedFiltersRef.current[data]?.includes(key),
+      [filters, setFilters] = React.useState(col.cellType?.tag
```

Comment:
Would go for `menuFilters` or something like that so that reader is sure which ones are meant.

Reply:
Indeed, surely more meaningful.